src/Tribe/Importer/Admin_Page.php

```diff
@@ -20,6 +20,39 @@ public function register_admin_page() {
 		);
 	}
 
+	public function add_settings_fields() {
+		$fields = array(
+			'csv-title' => array(
+				'type' => 'html',
+				'html' => '<h3>' . esc_html__( 'CSV Import Settings', 'the-events-calendar' ) . '</h3>',
+			),
+			'tribe-form-content-start' => array(
+				'type' => 'html',
+				'html' => '<div class="tribe-settings-form-wrap">',
+			),
+			'imported_post_status[csv]' => array(
+				'type' => 'dropdown',
+				'label' => __( 'Default status to use for imported events', 'the-events-calendar' ),
+				'options' => Tribe__Events__Importer__Options::get_possible_stati(),
+				'validation_type' => 'options',
+				'parent_option' => Tribe__Events__Main::OPTIONNAME,
+			),
+			'tribe-form-content-end' => array(
+				'type' => 'html',
+				'html' => '</div>',
+			),
+		);
+		foreach ( $fields as $key => $field_args ) {
+			if ( 'imported_post_status[csv]' !== $key ){
+				$value = Tribe__Events__Main::getOption( $key, null );
+			} else {
+				$value = Tribe__Events__Importer__Options::get_default_post_status( 'csv' );
+			}
+
+			$field = new Tribe__Events__Field( $key, $field_args, $value );
```

Comment:
`$field` is overwritten with each iteration of this loop and ultimately isn't retained when the method returns: do we need to keep a reference and can we set up a member field for that if we do?